src/configs.rs

```diff
@@ -31,6 +39,13 @@ lazy_static! {
         AtomicI64::new(BLOOM_MEMORY_LIMIT_PER_FILTER_DEFAULT);
     pub static ref BLOOM_USE_RANDOM_SEED: AtomicBool = AtomicBool::default();
     pub static ref BLOOM_DEFRAG: AtomicBool = AtomicBool::new(BLOOM_DEFRAG_DEAFULT);
+    pub static ref BLOOM_FP_RATE_DEFAULT_STRING: String = BLOOM_FP_RATE_DEFAULT.to_string();
```

Comment:
Can you add an actual f64 Mutex above this? We need to add this and change the f64 mutex.

```suggestion
    pub static ref BLOOM_FP_RATE_F64: Mutex<f64> = Mutex::new(BLOOM_FP_RATE_DEFAULT);
    pub static ref BLOOM_FP_RATE_DEFAULT_STRING: String = BLOOM_FP_RATE_DEFAULT.to_string();
```

Reply:
Can you change this `&*configs::BLOOM_FP_RATE_DEFAULT.to_string()` to a string literal?

Reply:
So do you want me to use this directly:
```
["bloom-fp-rate", &*configs::BLOOM_FP_RATE, "0.01", ConfigurationFlags::DEFAULT, None, Some(Box::new(configs::on_string_config_set))],
["bloom-tightening-ratio", &*configs::BLOOM_TIGHTENING_RATIO, "0.5", ConfigurationFlags::DEFAULT, None, Some(Box::new(configs::on_string_config_set))],
```

and avoid using to_string and the string constant we have in configs?